src/backend/distributed/sql/downgrades/citus--9.5-1--9.4-1.sql

```diff
@@ -2,7 +2,27 @@
 
 SET search_path = 'pg_catalog';
 
-DROP FUNCTION create_citus_local_table(table_name regclass);
+-- Check if user have any citus local tables.
```

Comment:
What if we undistribute the citus local table in case of a downgrade?

Reply:
You mean doing that automatically ? We might not be able to do this due to some limitations like `undistribute_table` errors out if table is involved in a  foreign key relationship

Reply:
Yeah I was thinking doing that automatically. The user can see that error if we do that automatically. Though I am not sure if it okay to undistribute a table without users manually doing that (Also undistribute table might be slow for a big citus local table)